packages/office-ui-fabric-react/src/utilities/test/TestUtilities.ts

```diff
@@ -0,0 +1,28 @@
+import { ReactWrapper } from 'enzyme';
```

Comment:
maybe these should go into the utilities package. I'd like to reduce the number of utilities directly in fabric-react.